tutorials/Orchest.ipynb

```diff
@@ -0,0 +1,267 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 5,
+  "metadata": {
+    "kernelspec": {
+      "display_name": "Python 3",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.9.6"
+    },
+    "colab": {
+      "name": "Welcome to PyClarify",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "![Clarify Logo](https://global-uploads.webflow.com/5e81e464dad44d3a9a32d1f4/5ed10fc3f1ff8467f4466786_logo.svg)\n",
+        "<img src=\"https://uploads-ssl.webflow.com/5f031b98adc00651e28ef04b/6058a5f7b4c86c42885a2c2c_orchest-logo-no-padding.svg\" alt=\"Orchest Logo\" width=\"200\"/>\n",
+        "\n",
+        "**Welcome the tutorial about creating and deployment data pipelines combining Clarify and Orchest!**\n",
+        "\n",
+        "<img src=\"../media/orchest/orchestration.jpg\" alt=\"clarify orchest\" width=\"400\">\n",
```

Comment:
I don't think this will work on colab.

Reply:
Yes, I left it like that so that you would visualize while reviewing, but will change for a link in the main branch before merging.

Reply:
Alright 👍

Reply:
Once Alexia has reviewed and you have updated the img paths I will approve :)